app/views/shared/_messages.html.slim

```diff
@@ -2,4 +2,4 @@
   - flash.each do |name, msg|
     - if msg.present? && msg.is_a?(String)
       div class="alert alert-#{name}" role='alert'
-        = msg
+        = sanitize(msg)
```

Comment:
Its what the design and UX team provided me. I would guess its something they would like to test?